components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/mediators/AWSLambdaMediator.java

```diff
@@ -121,13 +123,19 @@ private InvokeResult invokeLambda(String payload) {
                 log.error("Missing AWS Credentials");
                 return null;
             }
-            AWSLambda awsLambda = AWSLambdaClientBuilder.standard()
-                    .withCredentials(credentialsProvider)
-                    .build();
+            // set invoke request
+            if (resourceTimeout < 1000 && resourceTimeout > 900000) {
```

Comment:
Could this be null?

Reply:
It's an integer. If not defined it'll be 0.